### PR TITLE
Add websocket support to the mqtt broker example

### LINKED ISSUE #95: Add MQTT over WebSocket support and routing for the mqtt.kafka examples

### PR DESCRIPTION
**Pull request by vordimous**
- Add Websocket support for the mqtt.broker.kafka example
- Add routing to the mqtt.broker.kafka example
- formatting and whitespace cleanup

fixes #95

### CODE REVIEW

File: mqtt.kafka.broker/zilla.yaml [hunk 1]
```diff
@@ -28,12 +37,45 @@ bindings:
...
+    vault: my_servers
+    options:
+      keys:
+        - localhost
+      sni:
+        - localhost
+    exit: north_ws_http_server
+
+  # WebSocket server
+  north_ws_http_server:
```
Comment (jfallows) [inline mqtt.kafka.broker/zilla.yaml:52]: This should just be `north_http_server`, with an additional
```
upgrade: websocket
```
for each route below to exit `north_ws_server`.

File: mqtt.kafka.broker/zilla.yaml [hunk 2]
```diff
@@ -57,9 +112,10 @@ bindings:
       bootstrap:
         - mqtt-messages
         - mqtt-retained
+        - mqtt-places
```
Comment (jfallows) [inline mqtt.kafka.broker/zilla.yaml:115]: Should `mqtt-devices` in routes above via `with`, and this `mqtt-places` be the same `kafka` topic?

**Review by jfallows — CHANGES_REQUESTED** [review]

File: mqtt.kafka.broker/zilla.yaml (hunk 1 above)
Comment (vordimous) [reply mqtt.kafka.broker/zilla.yaml:52]: Is `upgrade` a zilla property or a header? I can't find an example it in the docs or the other `ws` examples.

Comment (jfallows) [reply mqtt.kafka.broker/zilla.yaml:52]: `upgrade` is an http header used by WebSocket handshake.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Protocol_upgrade_mechanism#upgrading_to_a_websocket_connection

**Pull request merged**